From git-commits-head-owner@vger.kernel.org Fri Dec  8 18:09:02 2006
Date: Thu, 7 Dec 2006 18:03:26 GMT
Message-Id: <200612071803.kB7I3Q0K010027@hera.kernel.org>
From: Linux Kernel Mailing List <linux-kernel@vger.kernel.org>
To: git-commits-head@vger.kernel.org
Subject: r8169: phy program update
Content-Length: 1196
Lines: 33

Gitweb:     http://git.kernel.org/git/?p=linux/kernel/git/torvalds/linux-2.6.git;a=commit;h=12d86f682e8acad8555718dc7b0082590f2365d0
Commit:     12d86f682e8acad8555718dc7b0082590f2365d0
Parent:     d03902b8864d7814c938f67befade5a3bba68708
Author:     Francois Romieu <romieu@fr.zoreil.com>
AuthorDate: Thu Nov 2 00:24:52 2006 +0100
Committer:  Francois Romieu <romieu@electric-eye.fr.zoreil.com>
CommitDate: Mon Dec 4 00:42:45 2006 +0100

    r8169: phy program update
    
    This is commented out in Realtek's driver as well.
    
    Signed-off-by: Francois Romieu <romieu@fr.zoreil.com>
---
 drivers/net/r8169.c |    5 -----
 1 files changed, 0 insertions(+), 5 deletions(-)

diff --git a/drivers/net/r8169.c b/drivers/net/r8169.c
index 7438049..0b57050 100644
--- a/drivers/net/r8169.c
+++ b/drivers/net/r8169.c
@@ -1286,11 +1286,6 @@ static void rtl8169_hw_phy_config(struct
 	/* Shazam ! */
 
 	if (tp->mac_version == RTL_GIGA_MAC_VER_04) {
-		mdio_write(ioaddr, 31, 0x0001);
-		mdio_write(ioaddr,  9, 0x273a);
-		mdio_write(ioaddr, 14, 0x7bfb);
-		mdio_write(ioaddr, 27, 0x841e);
-
 		mdio_write(ioaddr, 31, 0x0002);
 		mdio_write(ioaddr,  1, 0x90d0);
 		mdio_write(ioaddr, 31, 0x0000);

